float32 certainty
geometry_msgs/Vector3 position
int8 role
